.github/actions/py-dependencies/requirements-dev.txt

```diff
@@ -1,9 +1,8 @@
 # Needed for chek and lint scripts
```

Comment:
Not introduced by this PR but it's a good chance to fix the typo (missing c) in check.